packages/integration-karma/test/component/lockerIntegration/LockerIntegration.spec.js

```diff
@@ -0,0 +1,15 @@
+import { createElement } from 'lwc';
+
+import LockerIntegration from 'x/lockerIntegration';
+
+describe('integration', () => {
+    describe('with locker', () => {
+        it('should support manual construction', () => {
+            const elm = createElement('x-secure-parent', { is: LockerIntegration });
+            document.body.appendChild(elm);
+            // Verifying that shadow tree was created to ensure the component class was successfull processed
```

Comment:
There are two ways this can fail.
1. `createElement` can fail to process the provided component class.
2. Once the host is connected to the dom, the engine can fail to run the lifecycle hooks of the component class. eg: render() in this testcase